src/components/catPics.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function CatPics(props) {
+  const { name, description } = props;
```

Comment:
Nice job using destructuring!

Fun tip here: You can also do it like this: 

`function CatPicts({name, description}) {`

React is super fun!